docs/root/configuration/listeners/network_filters/_include/wasm-filter-proxy.yaml

```diff
@@ -0,0 +1,25 @@
+static_resources:
+  listeners:
+    - name: main
```

Comment:
could you use least indent for yaml formatting please, it makes configs much more readable